cli_tools/common/utils/storage/buffered_writer_test.go

```diff
@@ -47,7 +47,7 @@ func TestCreateNewChunkOnFirstWrite(t *testing.T) {
 
 	data := []byte("This is a sample data to write")
 
-	buf := NewBufferedWriter(ctx, bufferSize, workerNum, mockGcsClient, oauth, prefix, bkt, obj)
+	buf := NewBufferedWriter(ctx, bufferSize, workerNum, mockGcsClient, oauth, prefix, bkt, obj, "GCEExport")
```

Comment:
The overview mentioned that you're changing the prefix: "Fixed by allowing setting proper err log prefix.", but here it looks like it's still `GCEExport`. Which prefix were you hoping to use?

Reply:
"GCEExport:" will work. Current released version uses "[export]".

Reply:
The key point is the line 128 above. For that logic branch, there is only a "log.Fatal(err)", which uses "[export]" as prefix.
There should be a fmt.Printf() to output the msg with "GCEExport:", just as what line 122 did.

Reply:
Meanwhile, we also don't want to hard-code "GCEExport" as prefix in buffered_writer, because this lib is "common" and is also used by aws importer.